share/zsh/site-functions/_brew

```diff
@@ -29,6 +27,12 @@ _brew_outdated_formulae() {
   outdated_formulae=(`brew outdated`)
 }
 
+__brew_command() {
+  local command=$1
```

Comment:
I picked this line, but please always quote uses of a variable in the parts you modified/extended. We prefer to do that for consistency and in this case also for correctness because we're dealing with user input. In this particular example: Use `"$1"` instead of `$1`.